src/sass/pages/index.scss

```diff
@@ -196,7 +197,7 @@
   }
 
   &__content {
-    @include columns(8);
+    @include columns(10);
```

Comment:
reviewing the new grid, the `roadmap__content` seems to be 12 columns width; isn't it?
https://drive.google.com/file/d/0B9dw17GoC_XKR2QwY1VMVkVjNXc/view